math/src/elliptic_curve/edwards/point.rs

```diff
@@ -0,0 +1,180 @@
+use crate::{
+    cyclic_group::IsGroup,
+    elliptic_curve::{
+        point::ProjectivePoint,
+        traits::{EllipticCurveError, FromAffine, IsEllipticCurve},
+    },
+    field::element::FieldElement,
+};
+
+use super::traits::IsEdwards;
+
+#[derive(Clone, Debug)]
+pub struct EdwardsProjectivePoint<E: IsEllipticCurve>(ProjectivePoint<E>);
```

Comment:
I have a doubt, why is this wrapper necessary?

Reply:
We share ProjectivePoints data structure and basic functions between Curve Families, and we extend it by implementing the operator add for each family